test/plugin/test_out_bigquery.rb

```diff
@@ -695,28 +652,27 @@ def test_format_with_insert_id
       insert_id_field uuid
       field_string uuid
     CONFIG
-    driver.instance.start
-    buf = driver.instance.format_stream("my.tag", [input])
-    driver.instance.shutdown
-
-    assert_equal expected, MessagePack.unpack(buf)
+    mock(driver.instance).insert("foo", [expected], nil)
+    driver.instance_start
+    driver.run do
+      driver.feed('tag', now, input)
+      driver.flush
```

Comment:
`flush` will be done automatically (and can be disabled by `driver.run(flush: false)`. see [here](https://github.com/fluent/fluentd/blob/master/lib/fluent/test/driver/output.rb#L38))